cdap-ui/app/hydrator/services/create/stores/config-store.js

```diff
@@ -1093,9 +1093,16 @@ class HydratorPlusPlusConfigStore {
           this.emitChange();
         },
         err => {
+          let message = err;
+          if (err && err.data || err.response) {
```

Comment:
`&&` is higher precendence than `||`, so if `err` is null I think this will give an error.